app/js/components/manageApps/ManageApps.jsx

```diff
@@ -682,7 +682,8 @@ export default class ManageApps extends React.Component {
       });
     }).catch(
       (error) => {
-        toastr.error(error);
+        toastr.error("There is no internet connection");
```

Comment:
This statement presumes that the only reason this operation could possibly fail is if there's a network error.

Passing a rejection callback into the then block of the axios request would ensure you've accounted for http errors (4xx & 5xx). Then you can be quite certain that only non-http errors fall to the catch block

You might also want to inspect the error object before making that conclusion

Reply:
@segunolalive Thanks for the idea